Maquetado/scss/login.scss

```diff
@@ -0,0 +1,83 @@
+@mixin flexBox {
+    align-items: center;
+    display: flex;
+    justify-content: center;  
+}
```

Comment:
This mixin could be reused many times. I would recommend creating a new file "layout.css" / "mixins.scss" / "common.scss" you name it, that will be exported by every scss file that need to implement that mixin